httpipfs.go

```diff
@@ -119,6 +131,27 @@ func (hi *HttpIpfs) ServeHTTP(res http.ResponseWriter, req *http.Request) {
 
 	selNode := unixfsnode.UnixFSPathSelectorBuilder(path.String(), dagScope.TerminalSelectorSpec(), false)
 
+	sig := RequestSignature{
+		RequestId: req.Header.Get("X-Request-Id"),
```

Comment:
probably need to gate this on whether this header is included; maybe if there's no request id then you don't get an attestation?